static-www/templates/layout.tt2

```diff
@@ -1,77 +1,320 @@
+            target="_blank"
+            class="flex cursor-pointer items-center rounded-full bg-white/80 px-5 py-3 text-sm font-medium shadow transition hover:bg-black hover:text-white">
+            <svg class="mr-2" width="22" height="22" viewBox="0 0 24 24">
+              <path fill="currentColor" d="m12 16l-5-5l1.4-1.45l2.6 2.6V4h2v8.15l2.6-2.6L17 11zm-6 4q-.825 0-1.412-.587T4 18v-3h2v3h12v-3h2v3q0 .825-.587 1.413T18 20z" />
+            </svg>
+            Export
+          </a>
+        </div>
+      </div>
     </nav>
 
-    <!-- Facebook Comments -->
-    <div id="fb-root"></div>
-    <script>(function(d, s, id) {
-      var js, fjs = d.getElementsByTagName(s)[0];
-      if (d.getElementById(id)) return;
-      js = d.createElement(s); js.id = id;
-      js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.9&appId=260322844448067";
-      fjs.parentNode.insertBefore(js, fjs);
-      }(document, 'script', 'facebook-jssdk'));
+    <!-- Hero Section -->
+    <header class="py-10 sm:py-12">
+      <div class="mx-auto flex max-w-7xl flex-col gap-2 px-4 sm:px-6 lg:px-8">
+        <div class="flex flex-col items-center gap-6">
+          <h1 class="text-2xl font-extrabold sm:text-3xl lg:text-4xl">[% title %]</h1>
+          [% IF children %][% children %][% END %]
+        </div>
+      </div>
+    </header>
+
+    <!-- Main Content -->
+    <main class="mx-auto max-w-7xl space-y-8 px-4 pb-12 sm:px-6 lg:px-8">
+      [% content %]
+      <section class="rounded-4xl bg-white/80 p-6 shadow-md">
+        <div class="mb-6 flex flex-col gap-2">
+          <h2 class="text-xl font-bold">Comments</h2>
+          <p class="text-sm text-black/70">Share your thoughts or questions about this TIF data and engage with the community through the comments section below.</p>
+        </div>
+        <div class="fb-comments" data-href="[% url %]" data-numposts="5" data-width="100%" aria-label="Comments section"></div>
+      </section>
+    </main>
+
+    <!-- Footer Content -->
+    <div class="mx-auto flex max-w-7xl items-center justify-center px-4 sm:px-6 lg:px-8">
+      <a href="https://github.com/opennebraska" target="_blank" rel="noopener noreferrer" aria-label="Open/Nebraska GitHub">
+        <img src="/src/logo.png" alt="Open/Nebraska logo" class="h-20 w-20 rounded-full shadow-lg" loading="lazy" />
+      </a>
+    </div>
+
+    <footer class="mx-auto flex max-w-7xl grid-cols-3 flex-col items-center justify-center px-4 py-6 sm:px-6 md:grid md:items-center md:justify-between md:py-12 lg:px-8">
+      <div class="mt-4 flex items-center justify-center gap-x-4 overflow-x-auto text-sm md:order-2 md:mt-0 lg:gap-x-6">
+        <a href="/about.html" class="transition hover:text-black/50">About</a>
+        <a href="https://github.com/opennebraska/tif/issues" target="_blank" class="transition hover:text-black/50">Issues</a>
+        <a href="https://www.nebraska.gov/policies/" target="_blank" class="transition hover:text-black/50">Policies</a>
```

Comment:
I think we want to delete this link. Open/Nebraska is not an official government organization. So the official `Nebraska.gov` policies really have nothing to do with us.

Reply:
> I think we want to delete this link. Open/Nebraska is not an official government organization. So the official `Nebraska.gov` policies really have nothing to do with us.

👍

Reply:
fixed [1ed3dda](https://github.com/opennebraska/tif/pull/57/commits/1ed3dda187d2e021516d7ef2cd0aa2b92e456efe)